tools/RAiDER/etc/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash --login
 set -e
 conda activate RAiDER
-exec python -u raider.py "$@"
+exec raider.py "$@"
```

Comment:
```suggestion
exec python -um RAiDER "$@"
```
@cmarshak `-m` tells python to invoke the `__main__.py` script of a *module* or *package*, so you need to use RAiDER package name. 
https://peps.python.org/pep-0338/

Reply:
Oh wait, no, since y'all buried the `__main__.py` script under `RAiDER.cli` you'll probably have to do:
```python
exec python -um RAiDER.cli "$@"
```

But I'm not sure it'll work.

Otherwise, if you want to invoke the `raider.py` script, you'll need to provide the actual path to the script as `python -u` accepts a script *path*. Something like
```sh
exec python -u $(which raider.py) "$@"
```
might likely work, but since this script is *only* for the docker container, I'd just hard-code it as you *know* the path in the container.

Reply:
Python command line options are described here:
https://docs.python.org/3/using/cmdline.html

Reply:
Thank you, Joe. I am sorry to disturb you. I knew you would know and I am grateful for clarifying this solution. I can explain my confusion on this particular rabbit hole:

- `pyproject.toml` and `setup.py` append paths for CLIs to use the python scripts from the command line to be used as `<script-name> ...`.
- `python ...` searches for packages in the *site_packages* of the environment. 

In all the references I was using the library/module name was the same as the CLI entrypoint.

Reply:
And both the suggestions you gave work. Thank you.